tests/v6/utils-e2e-v6.ts

```diff
@@ -55,6 +56,16 @@ export async function runE2ETest(
   const amountToFund = BigInt(srcAmount) * 2n;
   // add allowance override to Augustus
   if (srcToken.toLowerCase() !== ETHER_ADDRESS) {
+    if (overrideBalance) {
```

Comment:
Let's add balance by default as it's used for all cases, no need in specifying it separately